ExpenseSplit/HomeScreen.swift

```diff
@@ -0,0 +1,162 @@
+//
+//  HomeScreen.swift
+//  ExpenseSplit
+//
+//  Created by Ankita Katoch on 8/9/24.
+//
+
+import Foundation
+import SwiftUI
+
+struct Friend: Identifiable {
+    let id = UUID()
+    let name: String
+    let avatar: String
+    let balance: String
+    let isOwed: Bool
+}
+
+struct HomeScreen: View {
+    @State private var showSettleBill = false
+        @State private var showCalculateTip = false
+    let friends = [
+        Friend(name: "Elephany", avatar: "elephant", balance: "€92.21", isOwed: false),
+        Friend(name: "Girrafy", avatar: "Girraf", balance: "£20.00", isOwed: true),
+        Friend(name: "Simba", avatar: "Simba", balance: "settled up", isOwed: false),
+        Friend(name: "Jerry", avatar: "jerry", balance: "$17.51", isOwed: false),
+        Friend(name: "Timothy", avatar: "tim", balance: "$87.28", isOwed: true),
+        Friend(name: "Pumba", avatar: "pumba", balance: "₹1,550.50", isOwed: true)
+    ]
+    
+    
+    var body: some View {
+        NavigationView {
```

Comment:
Your implementation looks great, and it’s clear you’ve put a lot of thought into organizing your views! I noticed that this view is a child view that’s already within a NavigationStack. Since the NavigationStack already handles navigation, the extra NavigationView inside this child view isn’t necessary and can be removed. This will help simplify the code a bit and avoid any potential navigation conflicts.